gnucash/gtkbuilder/assistant-stock-transaction.glade

```diff
@@ -0,0 +1,941 @@
+        <property name="can-focus">False</property>
+        <property name="border-width">6</property>
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="GtkLabel" id="stock_amount_title">
+            <property name="visible">True</property>
+            <property name="can-focus">False</property>
+            <property name="margin-bottom">6</property>
+            <property name="label" translatable="yes">Enter the date and the number of shares you gained or lost in the transaction.</property>
+            <property name="wrap">True</property>
+          </object>
+          <packing>
+            <property name="expand">False</property>
+            <property name="fill">True</property>
+            <property name="position">0</property>
+          </packing>
+        </child>
+        <child>
+          <!-- n-columns=2 n-rows=3 -->
+          <object class="GtkGrid" id="stock_amount_table">
+            <property name="visible">True</property>
+            <property name="can-focus">False</property>
+            <property name="border-width">6</property>
+            <property name="row-spacing">6</property>
+            <property name="column-spacing">6</property>
+            <child>
+              <object class="GtkLabel" id="stock_amount_label">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="halign">end</property>
+                <property name="label" translatable="yes">_Shares</property>
+                <property name="use-underline">True</property>
+                <property name="justify">center</property>
+              </object>
+              <packing>
+                <property name="left-attach">0</property>
+                <property name="top-attach">1</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLabel" id="prev_balance_label">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="halign">end</property>
+                <property name="label" translatable="yes">Previous Balance</property>
+                <property name="use-underline">True</property>
+                <property name="justify">center</property>
+              </object>
+              <packing>
+                <property name="left-attach">0</property>
+                <property name="top-attach">0</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLabel" id="prev_balance_amount">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="halign">end</property>
+                <property name="label" translatable="yes">(Previous Balance)</property>
```

Comment:
Is there no better solution here and later than parenthesis, perhaps some markup?

Reply:
This label is overwritten during code. So translation must be disabled. Thanks.

Reply:
Then you should change the `translatable` attribute value to "no" and add a comment `<!-- label will be overwritten in code -->`

Reply:
> Then you should change the `translatable` attribute value to "no" and add a comment `<!-- label will be overwritten in code -->`

In my experiment with `ninja pot` removing the `translatable` attribute was sufficient to avoid an entry in gnucash.pot.

Reply:
Of course. But it doesn't communicate to our future selves that the string is untranslated on purpose.